packages/math-input/src/components/tabbar/item.js

```diff
@@ -34,7 +36,7 @@ const styles = StyleSheet.create({
         boxSizing: "border-box",
         border: `1px solid transparent`,
         borderRadius: 2,
-        dislay: "flex",
+        display: "flex",
```

Comment:
I wonder if I should leave this typo in (or delete the line entirely). Obviously this has been not actually applying `display: "flex"` for a while... so I'm not sure this'll cause a regression by fixing it.

Reply:
I guess we'll find out when QE regression tests things.   Was this caught by Flow?